common/source/docs/common-optical-flow-sensor-setup.rst

```diff
@@ -14,6 +16,32 @@ With the sensor connected to the autopilot, connect to the autopilot with the Mi
 .. image:: ../../../images/PX4Flow_CheckForData_MP.png
     :target: ../_images/PX4Flow_CheckForData_MP.png
 
+If the vehicle does not appear on the GCS map, please check the following:
+
+- Check the rangefinder is operating correctly (see below)
+- Use the Ground Station to set the EKF origin.  If using Mission Planner, right mouse button click on the map, select "Set Home here", and choose, "set EKF origin here"
+
+Once the vehicle appears on the map, move the vehicle horizontally and confirm the vehicle icon on the map moves accordingly
+
+[site wiki="plane,copter"]
+Rangefinder Check
+=================
+
+Check the rangefinder's reported distance is between the :ref:`RNGFND1_MIN <RNGFND1_MIN>` and :ref:`RNGFND1_MAX <RNGFND1_MAX>` parameter values when on the ground and when held up
+
+.. image:: ../../../images/mp_rangefinder_lidarlite_testing.jpg
+    :target: ../_images/mp_rangefinder_lidarlite_testing.jpg
+    :width: 250px
+
+You may also download an onboard log and check the ``EKF5.meaRng`` message to confirm
+
+#. There is continuous range measurement
```

Comment:
Might need a comment here about the special case of 0.

Reply:
So I think min->max should set the usable range and RNGFND1_GNDCLEAR should set the value on the ground (which is 0 in my case)